bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/handler/VenstarThermostatHandler.java

```diff
@@ -186,7 +205,28 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 log.debug("Setting away mode to  {}", value);
                 setAwayMode(value);
                 updateIfChanged(CHANNEL_AWAY_MODE_RAW, new StringType(value.toString()));
+            } else if (channelUID.getId().equals(CHANNEL_FAN_MODE)) {
+                VenstarFanMode value;
+                if (command instanceof StringType) {
+                    value = VenstarFanMode.valueOf(((StringType) command).toString().toUpperCase());
+                } else {
+                    value = VenstarFanMode.fromInt(((DecimalType) command).intValue());
+                }
+                log.debug("Setting fan mode to  {}", value);
+                setFanMode(value);
+                updateIfChanged(CHANNEL_FAN_MODE_RAW, new StringType(value.toString()));
+            } else if (channelUID.getId().equals(CHANNEL_SCHEDULE_MODE)) {
```

Comment:
This is better to reverse and put the constant at left.

Reply:
I don't understand what you mean. This follows the format of the other similar blocks in the code.

Reply:
Ok, keep it as it is